packages/sample/src/HubStringify.ts

```diff
@@ -0,0 +1,17 @@
+import { RevHub } from "@rev-robotics/expansion-hub";
+
+export async function toString(hub: RevHub): Promise<string> {
```

Comment:
In Typescript and Javascript, there can only be one function implementation with a given name in scope in a given file. Any file that imports this function can't have its own `toString()` function, so this unfortunately needs a more descriptive name.

Reply:
Also, why is this function async?

Reply:
> This code is in the sample. It should never be imported by end users.

I know, but it's still a flaw within the sample codebase IMO, just one that doesn't matter as much

> getModuleAddress was originally async if I remember correctly. I think we can remove the async now.

Please do so

Reply:
Resolved both